### PR TITLE
Updated search bar making it visible and aligned with the web design

### PR DESCRIPTION
**Pull request by PraveenUppar**
# Pull Request Template for GSoC 2024

## Related Issue

- Updated search bar making it visible and aligned with the web design
- closes:#70

## Description of Changes

- Search Bar Redesign: Improved the visual design of the search bar to enhance user experience and alignment with the overall website design.
## Type of Pull Request

- [X] Bug Fix
- [ ] New Feature
- [ ] Performance Improvement
- [ ] Documentation Update
- [ ] Code Refactoring
- [ ] Security Enhancement
- [X] Other (Specify): UI /UX

## Screenshots / Demo (if applicable)

Screenshot 2024-10-05 235443

## Checklist for Contributor

- [X] I have reviewed my code before submission.
- [X] I have read and followed the project's Contribution Guidelines.
- [X] I have tested the code thoroughly.
- [X] I have attached related issue numbers, screenshots, or videos (if applicable).
- [X] I have added comments to my code where necessary for clarity.
- [X] I have checked the existing code base and my changes do not introduce any breaking changes.
- [X] My PR addresses only one issue; I have created separate PRs for other issues.
- [X] I have ensured that the PR does not include unrelated files.
- [X] I am mindful of the project's coding standards and best practices.

## Additional Information
N/A
## Contribution Guidelines Reminder

Thank you for contributing to our project! Please ensure you follow the guidelines below:

### Contribution Levels

2. **Task Categories:**
   - **Basic Tasks:** Primarily focused on small changes like minor bug fixes, documentation improvements, and unit testing. Labeled as **Easy**.
   - **Intermediate Tasks:** Moderate-level tasks that involve enhancing existing features, code refactoring, or performance improvements. Labeled as **Medium**.
   - **Advanced Tasks:** Complex tasks like building new features, significant refactoring, or critical bug fixes. Labeled as **Hard**.

3. **Acknowledging Effort:**
   - Exceptional contributions will be recognized with a higher contribution level if substantial effort and innovation are demonstrated.

### Review Process

- PRs will be reviewed by the maintainers, and feedback will be provided. Contributors should address feedback promptly and resubmit the PR if necessary.

- Contributors are encouraged to participate in the review process of other PRs for community growth.

We appreciate your time and effort in improving this project and look forward to your valuable contributions!

### CODE REVIEW

**Comment by PraveenUppar** [comment]
I have made changes in the search bar once check

**Review by dhairyagothi — COMMENTED** [review]
update your fork repo first

**Comment by PraveenUppar** [comment]
I have synced my fork once check if any changes just let me know

File: frontend/src/Pages/hamburger.jsx [hunk 1]
```diff
@@ -121,25 +121,18 @@ const SearchContainer = styled.div`
   position: fixed;
   top: 10px;
   right: 10px;
-  background-color: rgba(0, 0, 0, 0.5);
```
Comment (dhairyagothi) [inline frontend/src/Pages/hamburger.jsx:124]: @PraveenUppar  change it to whitish background

File: frontend/src/Pages/hamburger.jsx [hunk 2]
```diff
@@ -121,25 +121,18 @@ const SearchContainer = styled.div`
...
-  padding: 5px 10px; 
-  transition: width 0.4s ease; 
+  background-color: rgba(0, 0, 0, 0.5);
+  border-radius: 25px;
+  padding: 5px 10px;
 `;
 
 const SearchInput = styled.input`
   display: block;
-  padding: 8px;
```
Comment (dhairyagothi) [inline frontend/src/Pages/hamburger.jsx:132]: add this animations

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

File: frontend/src/Pages/hamburger.jsx (hunk 2 above)
Comment (dhairyagothi) [reply frontend/src/Pages/hamburger.jsx:132]: in ` const SearchInput = styled.input` `
```
  width: ${({ show }) => (show ? '200px' : '0px')};
  transition: width 0.4s ease;
    opacity: ${({ show }) => (show ? 1 : 0)};
  pointer-events: ${({ show }) => (show ? 'auto' : 'none')};
```

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Review by PraveenUppar — COMMENTED** [review]
1. I have changed the background colour whitish
2. Added the animation in the const SearchInput = styled.input

**Review by dhairyagothi — APPROVED** [review]

**Pull request merged**